charts/pega/values.yaml

```diff
@@ -114,6 +114,10 @@ global:
       # initialHeap: "4096m"
       # maxHeap: "7168m"
 
+      # Optionally provide runAsUser to override default pegauser
```

Comment:
By default ```pegasystems/pega``` image uses ```pegauser(9001)``` as the user; to accomplish {- give an example user case when the default really needs to change}, specify the alternative. For example, to {do this}, specify {what???}.

Reply:
you can address these documentation changes later using BUG-622868. All set!